.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "spack"]
   path = spack
-  #url = https://github.com/spack/spack
-  #branch = develop
+  url = https://github.com/spack/spack
```

Comment:
Is this correct? Why two URLs for the submodule?

Reply:
No, that is wrong. Thanks for catching this!